packages/app-page-builder/src/editor/plugins/elements/media/iframe/IFrameSettings.tsx

```diff
@@ -0,0 +1,97 @@
+import React from "react";
+import { css } from "emotion";
+import { merge } from "dot-prop-immutable";
+import { Form } from "@webiny/form";
+import { validation } from "@webiny/validation";
+import { withActiveElement } from "~/editor/components";
+import { DelayedOnChange } from "~/editor/components/DelayedOnChange";
+import { useEventActionHandler } from "~/editor/hooks/useEventActionHandler";
+import { UpdateElementActionEvent } from "~/editor/recoil/actions";
+import { PbEditorPageElementSettingsRenderComponentProps, PbEditorElement } from "~/types";
+// Components
+import Accordion from "../../../elementSettings/components/Accordion";
+import Wrapper from "../../../elementSettings/components/Wrapper";
+import InputField from "../../../elementSettings/components/InputField";
+
+const classes = {
+    gridClass: css({
+        "&.mdc-layout-grid": {
+            padding: 0,
+            marginBottom: 24
+        }
+    }),
+    gridCellClass: css({
+        justifySelf: "end"
+    })
+};
+
+const urlRegex =
+    /(https?:\/\/(?:www\.|(?!www))[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\.[^\s]{2,}|www\.[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\.[^\s]{2,}|https?:\/\/(?:www\.|(?!www))[a-zA-Z0-9]+\.[^\s]{2,}|www\.[a-zA-Z0-9]+\.[^\s]{2,})/gi;
+
+interface LinkSettingsFormData {
+    url?: string;
+}
+
+interface LinkSettingsPropsType {
+    element: PbEditorElement;
+}
+const LinkSettingsComponent: React.FC<
+    LinkSettingsPropsType & PbEditorPageElementSettingsRenderComponentProps
+> = ({ element, defaultAccordionValue }) => {
+    const handler = useEventActionHandler();
+
+    const { url } = element.data?.["iframe"] || {};
+
+    const updateElement = (element: PbEditorElement) => {
+        handler.trigger(
+            new UpdateElementActionEvent({
+                element,
+                history: true
+            })
+        );
+    };
+
+    const updateSettings = (data: LinkSettingsFormData) => {
+        // Skip update if nothing is changed.
+        if (data.url === url) {
+            return;
+        }
+
+        const isValidUrl = data.url && urlRegex.test(data.url);
```

Comment:
Let's use the `@webiny/validation` package here, which is already imported in this file.

```
const isValidUrl = data.url && validation.validateSync(data.url, "url:allowHref");
```

Reply:
Oh nice one thanks